README.md

```diff
@@ -213,18 +213,33 @@ Sending SQL: "SELECT playerName, sum(runs) AS sum_runs FROM "baseballStats" WHER
 [(19790, 581.0), (19977, 522.0), (19690, 520.0), (19805, 481.0), (20409, 410.0), (21171, 385.0), (19930, 378.0), (20355, 377.0), (19393, 326.0), (20437, 268.0)]
 ```
 
+## Development
+
+In order to develop this library, you need to have installed Poetry and tox.
```

Comment:
i am not 100% familiar with poetry here are several questions I have switching to it
1. is it 100% compatible in terms of package management comparing with pip / setup.py, e.g. all the things we do today can be done in poetry?
2. how much flexibility we can do for running a release? (e.g. running customized scripts / steps, etc) i will anticipate this would be super good via changing Makefile

Reply:
1. Yes and no, it depends. tl;dr: It's compatible in terms of installing and publishing packages in a sense that you can use pip to install packages generated with Poetry, but it's not compatible when doing the development and packaging itself (before releasing). `setup.py` is a reminiscent of distutils, which is [deprecated and is going to be removed with Python 3.12](https://docs.python.org/3/library/distutils.html). distutils and setuptools use that file for packaging and distributing Python packages. However, another approach to packaging Python libraries has been created, it's more modern and was [accepted as a PEP](https://peps.python.org/pep-0518/), and it's the approach that Poetry uses.
2. This matter is independent of Poetry, I think; I don't know of anything that older packaging libraries do that Poetry can't do, so I think that a sane mix of Poetry with the Makefile should be good enough. What do you think?

Reply:
Sorry, let me add something else which is relevant for clarification: Poetry here is being used only at development phase. As soon as the library is released to PyPI, Poetry is not involved anymore, users don't have to install Poetry to use the library. So in the end we would have an installable library no matter what tool we used under development, and without requiring the installation of the packaging tool that was used.

Reply:
Correct, tox is only a test runner, not a library. I've added it just so that we can run tests on multiple Python versions on our own computers (I run tests all the time, so this helps me keep things under control). If you prefer, I can remove it, and then we could keep using only pytest directly (instead of being called from tox).

Reply:
Not necessary. Just checking sounds great.